recipes/src/main/resources/quarkus-updates/core/3alpha.yaml

```diff
@@ -1229,6 +1233,10 @@ recipeList:
       oldPropertyKey: quarkus.openshift.group
       newPropertyKey: quarkus.openshift.part-of
       fileMatcher: "**/application*.yml"
+  - org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.jib.labels
+      newPropertyKey: quarkus.container-image.labels
+      fileMatcher: "**/application*.yml"
 
 #####
 # Adjust properties in application.yaml
```

Comment:
Add the same rule here, but with `fileMatcher: "**/application*.yaml"` (yaml instead of yml)

Reply:
Oh, I didn't see that

Reply:
Me neither, I just realized after I created https://github.com/quarkusio/quarkus-updates/pull/20 😝

Reply:
😆